shell/platform/fuchsia/flutter/vsync_waiter.cc

```diff
@@ -50,6 +52,17 @@ VsyncWaiter::VsyncWaiter(std::string debug_label,
             std::make_unique<fml::WeakPtrFactory<VsyncWaiter>>(this);
       }));
   session_wait_.set_handler(wait_handler);
+
+  std::string json_string;
```

Comment:
I think a better pattern would be read this on engine startup in `component.cc`, create a `FlutterRunnerSettings` object of some kind after parsing the json and pass the settings object to the components that require it.

Reply:
How about creating it in `session_connection.cc`? This seems to be the lowest common ancestor for everything that may need Fuchsia+Scenic product-specific offsets.

Reply:
Given that this configuration file can be used for potentially non-scenic specific configurations, e.g. image cache size or vulkan validation layer configuration, it might make sense to move it higher up.

Reply:
Done